docs/en_us/platform_api/source/user/overview.rst

```diff
@@ -0,0 +1,28 @@
+################################################
+EdX Platform User API Overview
+################################################
+
+Use the edX Platform User API you to view and update account information.
+
+The edX Platform User API uses Representational State Transfer (REST) design
+principles and supports the JavaScript Object Notation (JSON) data-interchange
+format. Our REST API is simple, lightweight and optimized.
+
+You can use the edX Platform User API for web, desktop, and mobile
+applications.
+
+*************************************
+EdX Platform User API Version 1
```

Comment:
This should say version 0. In fact, we probably should be saying the minor and patch version numbers too. How do we manage that, and how do the version numbers flow through from the code to the doc?